services/motion/builtin/builtin_test.go

```diff
@@ -1115,8 +1136,8 @@ func TestMoveOnGlobe(t *testing.T) {
 	ctx := context.Background()
 	// Near antarctica 🐧
 	gpsPoint := geo.NewPoint(-70, 40)
-	dst := geo.NewPoint(gpsPoint.Lat(), gpsPoint.Lng()+1e-5)
-	expectedDst := r3.Vector{X: 380, Y: 0, Z: 0} // Relative pose to the starting point of the base; facing north, Y = forwards
+	dst := geo.NewPoint(gpsPoint.Lat(), gpsPoint.Lng()+0.00007)
+	expectedDst := r3.Vector{X: 2662.16, Y: 0, Z: 0} // Relative pose to the starting point of the base; facing north, Y = forwards
```

Comment:
these numbers seem somewhat random, but I think thats fine.  Just want to double check that you did the math for these to validate that the conversion makes sense

Reply:
i think the conversion does make sense. Instead of doing `1e-5` we now do `7e-5` so that we go from distance of `380` to `>26000` since that is the default plan dev mm for move on globe requests

https://go.dev/play/p/GcDOGOw4bEs